.mvn/maven-build-cache-config.xml

```diff
@@ -21,9 +21,9 @@
     <!--
         Maven build cache configuration for aiSSEMBLE
 
-        This feature requires Maven 3.9+ and is currently disabled by default while we work out the kinks. To use it,
-        you must pass the -Dmaven.build.cache.enabled=true flag to your build. It is also useful to look over the other
-        build options available: https://maven.apache.org/extensions/maven-build-cache-extension/parameters.html
+        This feature requires Maven 3.9+. To use it, you must pass the -Dmaven.build.cache.enabled=true flag to
```

Comment:
Q: Is this right? Thought it was enabled by default and you have to disable it with `-Dmaven.build.cache.enabled=false`

Reply:
Good catch.  Fixed.